### ISSUE TITLE
Proposal: Option to disable volumes_from when using fig up

### ISSUE THREAD

**Issue by hamiltont** [docker/compose#984]
If I grok fig correctly, when I use fig up I essentially get the volumes from a prior container (assuming that one exists). 

I'm mounting volumes on my host using things like sshfs, and then using fig to expose those inside docker containers. I keep having to run `fig stop && fig rm --force && fig up -d` to actually get a fresh mount when the status changes on my host. For example, if sshfs crashes, then I can unmount and remount it on the host and it works on the host. Inside the docker container it's still not working at this point, but if I just stop and restart the container using fig, the mount point inside fig still acts like it's crashed. The only way to get it to work properly inside docker seems to get a proper bind mount from the original host location

Basically, if there was some option like `--no-volumes-from` on `fig up` that forced mounting from the original filesystem, I think that would be useful

**Comment by dnephin** [comment]
I think this would be fixed by #858. It won't use `volumes_from` any more, and instead only copies over non-host volumes.

**Comment by hamiltont** [comment]
Agreed, that should be sufficient. Thanks!

**Comment by funkyfuture** [comment]
@hamiltont Please close this issue then. Thanks.

**Comment by hamiltont** [comment]
Thanks for reminding me :)
